artifacts/EthereumTokens.json

```diff
@@ -0,0 +1,127 @@
+{
+	"chain": {
+		"1": "Mainnet",
+		"42": "Kovan"
+	},
+
+	"tokens": [
+		{
+			"symbol": "ETH",
+			"decimals": 18,
+			"chainId": 1,
+			"address": "0x0000000000000000000000000000000000000000"
+		},
```

Comment:
can we add ETH for Kovan as well?

Also WETH & USDC are genericAssets on Nikau (id 17003, 17004) can we either add these to Kovan or remove them from .env?

If keeping WETH we need SVG (copy & rename eth.svg?)

Reply:
good call, doing it now

Reply:
done